includes/definitions/sslvis.yaml

```diff
@@ -0,0 +1,14 @@
+os: sslvis
+text: 'Blue Coat SSL Visibility'
```

Comment:
What is this device?

Reply:
This is a new appliance formed as part of Blue Coat's portfolio when they purchased Netronome Syetems in 2016.

Reply:
Any links to it online?

Reply:
@laf Sure - https://www.symantec.com/products/ssl-visibility-appliance